taxi_service/settings.py

```diff
@@ -44,6 +44,9 @@
     "django.contrib.staticfiles",
     "debug_toolbar",
     "taxi",
+    "crispy_bootstrap4",
+    "crispy_forms",
+    # "debug_toolbar",
```

Comment:
The `debug_toolbar` is listed twice in the `INSTALLED_APPS` list, once commented and once uncommented. Please remove the commented entry to avoid redundancy.